spec/toy_robot_spec.rb

```diff
@@ -17,6 +17,36 @@
     end
   end
 
+  describe 'valid_place' do
+    subject { ToyRobot.new }
+    specify 'that when x value is between 0 and 4 and y value is between 0 and 4, it will return true' do
+      x = 0
+      y = 0
+      expect(subject.valid_place(x,y)).to be true
+    end
+
+    specify 'that when x value is not between 0 and 4 and y value is not between 0 and 4, it will return false' do
+      x = -1
+      y = 5
+      expect(subject.valid_place(x,y)).to be false
+    end
+  end ## end of describe 'valid_place'
+
+  describe 'replace_movements' do
```

Comment:
replace_movements is not a public method. Make it a private method, this means you can't test it. This is one of the things I find hardest about designing interfaces and testing.
However, in this case there is no logic to worry about and the code is only called in one place so I would get rid of the method and just put the code back in the place function. 

Just test it as another assertion on the place test.